src/locale/locales/fr/messages.po

```diff
@@ -6840,16 +6590,12 @@ msgstr "Pour signaler une conversation, veuillez signaler un de ses messages via
 
 #: src/view/com/composer/videos/SelectVideoBtn.tsx:106
 msgid "To upload videos to Bluesky, you must first verify your email."
-msgstr ""
+msgstr "Pour télécharger des vidéos sur Bluesky, vous devez d’abord vérifier votre e-mail."
```

Comment:
This one is fascinating, because I thought `télécharger` meant download, although I should have looked to see if `upload` has already been translated in another string (it probably has been, sorry).

But DeepL was **very** insistent that that was how `upload` (and related words) should be translated; I don't remember it ever showing me `envoyer` as an option.

Anyway, I promise to never, ever use `téléverser`, and `envoyer` does seem much clearer.